src/api/getSingleMessage.js

```diff
@@ -0,0 +1,12 @@
+import { apiGet } from './apiFetch';
+
+export default async (
+  auth,
+  messageId,
+) =>
+  apiGet(
+    auth,
+    `messages/${messageId}`,
+    {},
+    res => res,
```

Comment:
I have a question here: Would it be better to change this to `res => res.raw_content` since we are probably only interested in that field of the end point?